chia/cmds/wallet_funcs.py

```diff
@@ -170,6 +173,167 @@ async def add_token(args: dict, wallet_client: WalletRpcClient, fingerprint: int
+        for item in [*offers, *requests]:
+            wallet_id, amount = tuple(item.split(":")[0:2])
+            if int(wallet_id) == 1:
+                name: str = "XCH"
+                unit: int = units["chia"]
+            else:
+                name = await wallet_client.get_cat_name(wallet_id)
+                unit = units["colouredcoin"]
+            multiplier: int = -1 if item in offers else 1
+            printable_dict[name] = (amount, unit, multiplier)
+            if uint32(int(wallet_id)) in offer_dict:
+                print("Not creating offer: Cannot offer and request the same asset in a trade")
+                break
+            else:
+                offer_dict[uint32(int(wallet_id))] = int(Decimal(amount) * unit) * multiplier
+        else:
+            print("Creating Offer")
+            print("--------------")
+            print()
+            print("OFFERING:")
+            for name, info in printable_dict.items():
+                amount, unit, multiplier = info
+                if multiplier < 0:
+                    print(f"  - {amount} {name} ({int(Decimal(amount) * unit)} mojos)")
+            print("REQUESTING:")
+            for name, info in printable_dict.items():
+                amount, unit, multiplier = info
+                if multiplier > 0:
+                    print(f"  - {amount} {name} ({int(Decimal(amount) * unit)} mojos)")
+
+            confirmation = input("Confirm (y/n): ")
+            if confirmation not in ["y", "yes"]:
+                print("Not creating offer...")
+            else:
+                offer, trade_record = await wallet_client.create_offer_for_ids(offer_dict, fee=fee)
+                if offer is not None:
+                    with open(pathlib.Path(filepath), "w") as file:
+                        file.write(bytes(offer).hex())
+                        file.close()
+                    print(f"Created offer with ID {trade_record.trade_id}")
+                    print(f"Use chia wallet get_offers --id {trade_record.trade_id} -f {fingerprint} to view status")
+                else:
+                    print("Error creating offer")
+
+
+def timestamp_to_time(timestamp):
+    return datetime.fromtimestamp(timestamp).strftime("%Y-%m-%d %H:%M:%S")
+
+
+async def print_offer_summary(wallet_client: WalletRpcClient, sum_dict: dict):
+    for asset_id, amount in sum_dict.items():
+        if asset_id == "xch":
+            wid: str = "1"
+            name: str = "XCH"
+            unit: int = units["chia"]
+        else:
+            result = await wallet_client.cat_asset_id_to_name(bytes.fromhex(asset_id))
+            wid = "Unknown"
+            name = asset_id
+            unit = units["colouredcoin"]
+            if result is not None:
+                wid = str(result[0])
+                name = result[1]
+        print(f"    - {name} (Wallet ID: {wid}): {Decimal(int(amount)) / unit}")
```

Comment:
When creating the offer, the amount in mojos is shown, while the summary view here omits it and potentially shows the amount in scientific notation:

```
Creating Offer
--------------

OFFERING:
  - 0.00000000001 XCH (10 mojos)
REQUESTING:
  - 0.111 ABC (111 mojos)
```
```
Summary:
  OFFERED:
    - XCH (Wallet ID: 1): 1E-11
  REQUESTED:
    - ABC (Wallet ID: 2): 0.111
```

```suggestion
        print(f"    - {name} (Wallet ID: {wid}): {Decimal(int(amount)) / unit} ({int(Decimal(amount))} mojos)")